### ISSUE TITLE
Pharmacy Reports 

### ISSUE THREAD

**Issue by Muchina-David** [chap-uzima/IQCare#50]
@chris-githu I kindly wish to know whether the reporting component of the Pharmacy Module has been finalized i.e. system generated FCDRR and FMAPS reports.

**Comment by chris-githu** [comment]
@Muchina-David Yes. I am working on these two reports. Targeting release of IQTools end of next week.

**Comment by Muchina-David** [comment]
@chris-githu Okay. Thanks.

**Comment by Muchina-David** [comment]
@chris-githu Once the anti-TB drugs are added it would be good to have a separate FCDRR for these drugs. I know a few drugs would appear in both reports. I will share with you the FCDRR for TB drugs.
